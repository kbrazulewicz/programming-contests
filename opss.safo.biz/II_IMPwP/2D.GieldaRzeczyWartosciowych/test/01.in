8
A K 100 10.00
A K 100 10.01
A K 100 10.02
A S 100 10.00
B S 100 10.00
B S 100 10.01
B S 100 10.02
B K 100 11.00
