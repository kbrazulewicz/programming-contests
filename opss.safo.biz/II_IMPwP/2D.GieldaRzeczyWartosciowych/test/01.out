2
A 100 10.02
B 100 10.00
